app/browser/select-dataset-step.tsx

```diff
@@ -472,6 +472,17 @@ const SelectDatasetStepContent = ({
                             <Icon name="arrowRight" size={20} />
                           )
                         }
+                        sx={
+                          odsIframe
+                            ? {
+                                backgroundColor: "#009688",
+
+                                "&:hover": {
+                                  backgroundColor: darken("#009688", 0.2),
+                                },
+                              }
+                            : null
```

Comment:
Nit: I guess it could be a Button color variant ?

Reply:
Technically yes, but as this color is only used in this place, I feel like adding it as a variant might be an overkill for now (I briefly tried but there were some errors related to our current logic that expects 50, 100, 200, etc). I'll leave it like this for now and add a comment 🙏